src/styles/3005.js

```diff
@@ -0,0 +1,145 @@
+import legend from './legends.js';
+
+const vectorTileStyling = {
```

Comment:
There's a lot of commonality between the various cases in many of the functions here, namely `{ fill: true, fillOpacity: 1, weight: 0 }` in many places. If that's not coincidence, it should be DRYed up in a single constant that is used in each case.

Reply:
Added a default style to reduce this repetition.